sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/TestSuiteBase.java

```diff
@@ -90,7 +90,7 @@
                                           allEqualOrLowerConsistencies(accountConsistency)));
         preferredLocations = immutableListOrNull(parsePreferredLocation(TestConfigurations.PREFERRED_LOCATIONS));
         protocols = ObjectUtils.defaultIfNull(immutableListOrNull(parseProtocols(TestConfigurations.PROTOCOLS)),
-                                              ImmutableList.of(Protocol.HTTPS, Protocol.TCP));
+                                              ImmutableList.of(Protocol.TCP));
```

Comment:
this is not the only place that we pass Direct Https.

Please see this:
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TestConfigurations.java#L81-L85

you may need to change the test yaml files which populate this environment variable.

Reply:
Yes, I will update the CI yml files as well. Thanks for catching this.

Reply:
Updated.